### ISSUE TITLE
Create HPL Vuclan Shutdown call and workflow

### ISSUE THREAD

**Issue by simonfernandes** [UserOfficeProject/user-office-project-issue-tracker#824]
We need to create a call with a corresponding workflow for the skinny Vulcan HPL round.

This will include working out with HPL things such as the call dates, reference number format, workflow, submission message (etc.).

**Comment by Cat-Lucifer** [comment]
@simonfernandes Is this for the HPL call proper (i.e. not the upcoming skinny Vulcan shutdown call)?

**Comment by simonfernandes** [comment]
@Cat-Lucifer I was meaning for the skinny call

**Comment by Cat-Lucifer** [comment]
Both questionnaire and workflow are already done.  Questionnaire to be demo-ed to HPL group tomorrow, so will be finalised thereafter.

@ACLay Was the new route Q key (vulcan_shutdown_access_route) sorted in R&A?

@simonfernandes Has reporting been tested yet (22810000)?

Just need to know so I can respond accordingly in the meeting tomorrow 🙂

**Comment by ACLay** [comment]
Changes in allocations are going through review. We'll still need to run a release once that's done.

If this issue is being rescoped from all HPL support to just the vulcan shutdown route, do we need to open another for the rest of HPL?

**Comment by simonfernandes** [comment]
New issue for main HPL call integration: https://github.com/isisbusapps/ISISBusApps/issues/924
New issue for main HPL call setup on submissions side: https://github.com/UserOfficeProject/user-office-project-issue-tracker/issues/922

--- 

@Cat-Lucifer We need to wait on the Allocations changes that Alex mentioned above before we test the reporting. As soon as that's ready we can start it this sprint and hopefully address any necessary changes quickly.

**Comment by Cat-Lucifer** [comment]
@simonfernandes @ACLay I expect that HPL will want this call to be opened by the end of this month, but I will confirm after the meeting tomorrow.

**Comment by Cat-Lucifer** [comment]
@simonfernandes @ACLay The demo in the HPL meeting went well!  The call is still being planned to open by the end of the month.  I am waiting on some info from CLF and once I have that the call will open.
